Makefile

```diff
@@ -1,14 +1,73 @@
+SHELL := /bin/bash
+
+NAME ?= terrahelp
+
 BUILDARGS ?= -mod=vendor
 
+BIN := $(CURDIR)/bin
+DIST := $(CURDIR)/dist
+OUPUT_FILES := $(BIN) $(DIST)
+
+PLATFORMS ?= darwin linux
+ARCH ?= amd64
+OS = $(word 1, $@)
+
 check:
 	go vet $(BUILDARGS) ./...
+.PHONY: check
```

Comment:
[MINOR] just-style guides and maybe a preference but can use phony targets before the targe declaration.
http://clarkgrubb.com/makefile-style-guide#phony-targets